shacl12-core/index.html

```diff
@@ -7373,6 +7373,25 @@ <h3>sh:name and sh:description</h3>
 					multiple <a>values</a>, but should only have one <a>value</a> per language tag.
 				</p>
 			</section>
+			<section id="codeIdentifier">
+				<h3>sh:codeIdentifier</h3>
+				<p>
+					Shapes may have one <a>value</a> for <code>sh:codeIdentifier</code>
+					to suggest a name that can be used for a representation of the shape in APIs, query languages
+					and similar programmatic access.
+					The values of <code>sh:codeIdentifier</code> are <a>literals</a> with <a>datatype</a> <code>xsd:string</code>
```

Comment:
Minor suggestion - because the previous text says "one value", "values" and "are literals" seems to hint otherwise:

```suggestion
					The value of <code>sh:codeIdentifier</code> is a <a>literal</a> with <a>datatype</a> <code>xsd:string</code>
```